fastapi_opa/opa/opa_middleware.py

```diff
@@ -19,9 +21,19 @@
 
 
 class OPAMiddleware:
-    def __init__(self, app: ASGIApp, config: OPAConfig) -> None:
+    def __init__(
+        self,
+        app: ASGIApp,
+        config: OPAConfig,
+        skip_endpoints: Optional[List[str]] = [
+            "/openapi.json",
+            "/docs",
+            "/redoc",
+        ],
+    ) -> None:
```

Comment:
To properly skip the graphql endpoint from any authentication, I had to extend the list of endpoints skipped by the middleware.